editor/editor_themes.cpp

```diff
@@ -246,7 +286,7 @@ void editor_register_and_generate_icons(Ref<Theme> p_theme, bool p_dark_theme =
 	const Color error_color = p_theme->get_color("error_color", "Editor");
 	const Color success_color = p_theme->get_color("success_color", "Editor");
 	const Color warning_color = p_theme->get_color("warning_color", "Editor");
-	dark_icon_color_dictionary[Color::html("#ff0000")] = error_color;
+	dark_icon_color_dictionary[Color::html("#ff5d5d")] = error_color;
```

Comment:
That's a third override for red as defined in lines 191 and 208, is that intended?

Reply:
I initially split the other two because they were used in different context. It was intentional, but given this override here is always applied, I guess there is simply no point.